lib/puppet/application/script.rb

```diff
@@ -0,0 +1,261 @@
+require 'puppet/application'
+require 'puppet/configurer'
+require 'puppet/util/profiler/aggregate'
+require 'puppet/parser/script_compiler'
+
+class Puppet::Application::Script < Puppet::Application
```

Comment:
What is the user value for this application? We have a ticket about deprecating/removing some applications and don't want to make the situation worse.

Reply:
This PUP- 7879 "Add a "puppet script" application that is similar to apply.". Is that one of the apps that should be removed? I was under the impression that what was in feature_puppet-tasks was going into master "as is". @hlindberg , is this redundant now?

Reply:
Just because it was approved and merged to the feature branch doesn't mean we want it in master. What purpose does the script application serve? There's no mention of it in the PUP-7879 ticket, nothing in acceptance criteria, or documentation.

Reply:
Is there anything keeping us from merging this and then address the ticket with removals once it's all on master?

Reply:
While we can remove this, it serves the purpose of being able to try out snippets of logic without using bolt CLI UI. The alternative is to use `puppet apply --tasks` which then also needlessly produce a catalog. Clearly it can be removed after the merge if we so desire.

Reply:
If it's bolt specific then you should put the 'Script' application in bolt's `boltlib`